config/content/Join.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable max-len */
+export default {
+  JoinUs: {
+    head:'Be a hack mentor',
+    img: {
+      src: 'https://res.cloudinary.com/dscnitrourkela/image/upload/project-oregano/assets/qt8zgt5yhmsdhqo5qux7.gif',
+      alt: 'Join Us',
+    },
+    heading: 'Be the mentor you were meant to be!',
+    description: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Delectus at provident',
```

Comment:
Either add relevant content here or remove it. And similarly below

Reply:
Noted.